docs/wip-plugins.md

```diff
@@ -85,6 +85,24 @@ Example:
 }
 ```
 
+### HTTP
+
+Allows the plugin to send HTTP requests.
+
+Example:
+
+```json
+{
+  ...,
+  "permissions": [
+    {
+      "type": "HTTP"
+    },
+    ...
+  ]
+}
+```
+
```

Comment:
I think the permission should be renamed to something along the lines of Network instead